Fletcher16.cpp

```diff
@@ -43,5 +61,19 @@ void Fletcher16::add(const uint8_t * array, uint16_t length)
 }
 
 
+uint16_t Fletcher16::getFletcher()
+{
+  if (_s1 >= FLETCHER_16) _s1 -= FLETCHER_16;
+  if (_s2 >= FLETCHER_16) _s2 -= FLETCHER_16;
```

Comment:
Or should we use here `255` instead of the constant?

Reply:
255 is just as constant as FLETCHER16 

If you want to replace ```#define FLETCHER16``` then it should become:
```cpp
const uint8_t FLETCHER16 = 255;
```
That would add explicit type information for the compiler